front/app/components/FormBuilder/components/FormBuilderToolbox/ToolboxItem.tsx

```diff
@@ -65,9 +68,21 @@ const ToolboxItem = ({
       // remove the role attribute when we add drag and drop functionality
       role="button"
       data-cy={rest['data-cy']}
+      disabled={disabled ? true : false}
```

Comment:
Since this is a boolean, we don't have to use a ternary operator

Reply:
I think this one might be a bit different, since disabled is either boolean or undefined :thinking: and the disabled variable on Box only accepts a boolean.

Reply:
Or would it be better to have:

`!isNilOrError(disabled) ? disabled : false` instead maybe?

Reply:
We can do `disabled={!!disabled}`

Reply:
Ah yes ok - I forget about "!!" sometimes. Need to remember to use that in these cases! Thank you!! :)